src/Module.cs

```diff
@@ -316,6 +316,43 @@ public static Module DeserializeFile(Engine engine, string name, string path)
             return new Module(handle, name);
         }
 
+        /// <summary>
+        /// Convert WAT (Web Assembly Text) into WASM bytes
+        /// </summary>
+        /// <param name="text"></param>
+        /// <returns></returns>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static byte[] Wat2Wasm(string text)
```

Comment:
I don't think we have to limit ourselves to the underlying C API name here. Perhaps `ConvertText` would be a better name for C# (open to other names too)?